packages/design-system/src/components/snackbar/snackbarContainer.tsx

```diff
@@ -157,57 +151,50 @@ function SnackbarContainer({
       }
     });
   }, [notifications, speak]);
-  const nodeRefs = useRef<Record<string, HTMLElement>>({});
   return (
     <StyledContainer placement={placement}>
       <TransitionGroup>
-        {orderedNotifications.map((notification, index) => {
-          const {
-            actionLabel,
-            dismissible,
-            message,
-            onAction,
-            preventActionDismiss,
-            preventAutoDismiss,
-            timeout,
-            ...notificationProps
-          } = notification;
-
-          const id = notification.id || ids[index];
-
-          return (
-            <CSSTransition
-              in
-              key={id}
-              timeout={300}
-              unmountOnExit
-              nodeRef={{ current: nodeRefs.current?.[id] }}
-              classNames="react-snackbar-alert__snackbar-container"
-            >
-              <ChildContainer
-                ref={(el) => {
-                  if (el) {
-                    nodeRefs.current[id] = el;
-                  }
-                }}
+        {orderedNotifications[0] &&
```

Comment:
For readability I think we should just keep the `.map()` here.

Makes it easier to use in case we change `max` again for some reason (even if just for debugging)